### DISCUSSION TITLE
React Native support

### DISCUSSION THREAD

**Discussion by Sergioloid** [tbleckert/react-select-search#155] [Ideas]
Hello guys,

I really love this library and I would like to use it in my React Native project.
Unfortunately, I've tried to import it without luck, receiving a bunch of errors.

Has anyone successfully used this lib in his React Native project?

If not, I think that adding react-native support should be a very beautiful idea to implement. Actually it seems that this lib was the best-looking and customizable one.

Thanks a lot for any reply.

**Comment by tbleckert** [comment]
Thanks for the support! I agree with you, would be great to get this functioning in React Native. I guess it already works in headless mode since we since the hooks doesn't render anything. The best approach here would be to build a separate component for RN that uses `useSelect`.

If anyone is interested, a great start would be to get a POC running and take it from there.
